src/DataTool.php

```diff
@@ -173,6 +173,29 @@ public function setFields($fieldDefs)
                 continue;
             }
 
+            // enum probability option
+            if (!empty($fieldRules['enum_key_probabilities'])
```

Comment:
The code should work for all `enum` fields regardless of whether some `enum_key_probabilities` are defined in the config. If `enum_key_probabilities` is empty then assign 100/count(keys) to every key. This way you would need to handle only one case in `handleType` instead of two (`if probabilities were used` and `use generic way`).